### PR TITLE
Fix nil resource issue

### PR DESCRIPTION
**Pull request by martinstiago**
This PR intends to fix the issue #23, when the action is not tracked when the resource is nil.

### CODE REVIEW

File: spec/tracker_spec.rb [hunk 1]
```diff
@@ -44,6 +45,16 @@
     expect(result).to match(['Here comes the test', action: 'Here comes the object test'])
   end
 
+  context 'when the resource does not exists' do
+    before(:each) do
+      allow_any_instance_of(described_class).to receive(:resource).and_return(Object.new)
```
Comment (antoniosb) [inline spec/tracker_spec.rb]: Isn't this stub intended to return `nil` ?

Comment (taiar) [inline spec/tracker_spec.rb]: Sure, @antoniosb. Fixed!

**Comment by taiar** [comment]
:+1:

**Comment by joaofraga** [comment]
LGTM :+1:

**Comment by antoniosb** [comment]
:loop: :+1:

**Pull request merged**